### ISSUE TITLE
Allow for user selection after they've been added

### ISSUE THREAD

**Issue by scottAnselmo** [jellyfin-archive/jellyfin-roku-legacy#310]
**Describe the feature you'd like**

There is an ability to:

1. 'Add a Profile' that allows input (i.e. server, uid, password). This is saved so that these credentials don't have to re-entered every time you want to change profiles (e.g. change from parent profile to kid)
2. Add an option to change profiles w/o re-entering server, uid, password, etc. The only thing that should be prompted is a pin if the profile is secured via a pin

**Additional context**

Basically at some point bring back the add user feature that was understandably removed for the time being:

https://github.com/jellyfin/jellyfin-roku/issues/192

Potentially encapsulate all in a menu like Netflix's pattern:

![image](https://user-images.githubusercontent.com/3056597/99910913-c8554b80-2ce8-11eb-9633-e8d9394b06b3.png)

**Comment by neilsb** [comment]
There's some ideas already in the works.  

![LoginConcept](https://user-images.githubusercontent.com/5013482/99940263-b49afb00-2d63-11eb-820e-cae3dde848d6.png)

Rather than having to add profiles, what I'm currently thinking is that each time you connect to a server, that server is "Saved" and available to select from a list.   Once a server has been selected it will show a list of all users that have been setup to show on the login screen (i.e. _Hide from Login Screen_ disabled) or the option to enter username/password manually.  Again, once a user has logged in "manually" from the login screen, this will be remembered and will be selectable from the list the next time.

There will be the option to _forget_ bother users and servers.   

Choosing to _Switch User_ or _Logout_ will take you back to the login screen to choose a new user.  From there you will be able to change to server.

**Comment by neilsb** [comment]
Also worth noting, at the current time if you remove the "Hide from Login Screen" option against a user, the Roku Client does show a list of users when logging in to allow you to select.  And if a _Short Pin_ is set against a user, this can be entered into the password field.   All this works in the current version.  (Thanks to #164)

**Comment by XaosTX** [comment]
Right now, the only issue that is keeping me from fully switching to Jellyfin is that I have multiple children at home that each have their own profile and need to be able to easily login/switch on our Roku TV.  Having to type in their usernames in addition to password/pin is too cumbersome for the little ones and the bigger ones. ;-)

**Comment by neilsb** [comment]
@XaosTX Just as a short term option, did you see the above comment by me about removing the "Hide from Login Screen" option on the user profiles, which will show a netflix style user list when not logged in?

**Comment by genebean** [comment]
I’d love to see this happen. It seems directly related to https://features.jellyfin.org/posts/239/allow-user-subprofiles-possibility too

**Comment by FirbyKirby** [comment]
Just adding my enthusiasm for this feature. The devs are doing a great job with this client, and this feature would really improve adoption in my home.

**Comment by dathbe** [comment]
Just a suggestion, but couldn't this be easily implemented by adding some code that:
1) allows multiple different logins to the same server (I haven't tested this, so that functionality may already be there)
2) _If_ there are multiple logins (whether they be to multiple servers OR multiple users on a single server OR a combination), the load page will be a selection page; if there is only a single login, the app would proceed to the home screen as it does now
3) The selection page would ideally be somewhat pretty, but functionality is the more important aspect, so even if it is a list like the Options>Change Server screen it would get the job done.

This would be really useful to prevent the "Watching" list from getting cluttered and messing up your place in series and such.

Thanks!

**Comment by kirk56k** [comment]
I'd like to put my 2 cents in for this idea as well. It's the only show-stopper preventing me from switching from Plex at the moment (for similar reasons to those above). I noticed that the Android TV app has the ability to "Always ask for user login at startup", and combine that with the ability to set a zero-length PIN for local network users, and it would have been a possibility (albeit still requiring more input than I'd like to switch, still possible for my 2nd grader). I understand that they are removing the ability to use PIN's at all in an upcoming version, however, so the ability to remember the credentials of multiple users and easily switch between them (or preferably prompt on every startup if there are more than one set of remembered credentials) would be ideal for this and likely all the other clients.
